packages/smooth_app/lib/pages/user_preferences_settings.dart

```diff
@@ -102,9 +103,42 @@ class UserPreferencesSettings extends AbstractUserPreferences {
         ),
         SmoothListTile(
           text: appLocalizations.support,
-          leadingWidget: const Icon(Icons.launch),
-          onPressed: () => LaunchUrlHelper.launchURL(
-              'https://slack.openfoodfacts.org/', false),
+          onPressed: () {
+            showDialog<void>(
+              context: context,
+              builder: (BuildContext context) => SmoothAlertDialog(
+                close: false,
+                body: Column(
+                  children: <Widget>[
+                    SmoothMainButton(
+                      important: false,
+                      text: appLocalizations.support_join_slack,
+                      onPressed: () {
+                        LaunchUrlHelper.launchURL(
+                          'https://slack.openfoodfacts.org/',
+                          false,
+                        );
+                      },
+                    ),
+                    const SizedBox(height: 20),
+                    SmoothMainButton(
+                      important: false,
+                      text: appLocalizations.support_via_email,
+                      onPressed: () async {
+                        final PackageInfo packageInfo =
+                            await PackageInfo.fromPlatform();
+                        LaunchUrlHelper.launchURL(
+                          // TODO(M123): Change subject name when we have a different app name
+                          'mailto:contact@openfoodfacts.org?subject=Smoothie%20help&body=Version:${packageInfo.version}',
```

Comment:
Shouldn't that be translated anyway?

Reply:
We can only translate `help` and `version` I don't think thats necessary,